packages/mui-material/src/Tooltip/Tooltip.test.js

```diff
@@ -1247,6 +1323,94 @@ describe('<Tooltip />', () => {
     });
   });
 
+  describe('prop: slots', () => {
```

Comment:
Is it possible to use `describeConformance`'s slots test suite? I remember there being some issues using it as the root slot is not handled in the same way in the Tooltip, is that why we need to add these separately here?

Reply:
Good point, I think moving forward with `slotProps`, the tests should be in `describeConformance`.

Reply:
Done.